Update README.md (#61)
Zinka010/cs348-project: README.md

- Justin Toft
- Justun Gao
- Amman Waheed

# Sources for the User Icons

> All of the pictures I used are from Flaticon, which are free for commercial and personal projects. I have listed them below.

Reviewer: ```suggestion
All of the team icons used are from Flaticon, which is free for commercial and personal projects. They are listed below.
```
